build/KoreBuild.sh

```diff
@@ -98,6 +98,16 @@ if [ ! -z "$KOREBUILD_SKIP_RUNTIME_INSTALL" ]; then
     # Add to the _end_ in case preferred .NET CLI is not in the default location.
     [[ ":$PATH:" != *":$DOTNET_INSTALL_DIR:"* ]] && export PATH="$PATH:$DOTNET_INSTALL_DIR"
 else
+    # Temporarily install these runtimes to prevent build breaks for repos not yet converted
```

Comment:
These should be moved down few lines, after DOTNET_INSTALL_DIR is set and dotnet-install.sh has been chmod-ed